docs/development.md

```diff
@@ -46,13 +35,11 @@ Node, NPM, and PNPM are well-behaved on Windows and Linux, and the difference in
 
 ## Setup: Configuration
 
-Wikijump has a lot of configuration options that can set. The primary configuration file can be found in `web/conf/`, named `wikijump.ini`. Freshly cloned, your Wikijump repo won't have this file. However, it will have an example configuration, `wikijump.ini.example`, which you can just copy and rename to `wikijump.ini` for the time being.
-
-Second, there is a Docker configuration file that configures the various containers that host Wikijump in the local environment. You can find this file in `install/local/dev/`, named `docker-compose.yaml`.
+There are a couple of configuration files that need to be initialized prior to running your instance of Wikijump. These will be the `config.toml` and `.env` files, both located in the `deepwell` subdirectory. Both of these files can be copied from their `.example` counterparts without changing them, though it is worth looking through them briefly to understand what can be configurated for your instance.
 
-Notice that in `docker-compose.yaml`, there are configuration options for the domains to use. For development purposes, these are set to `wikijump.localhost`. This is the domain you will be connecting to, e.g. `http://www.wikijump.localhost`. The TLD `.localhost` is just like the usual `localhost` domain. Your browser trusts both of these domains implicitly so there is no need to use `HTTPS`.
+There is also a Docker configuration file that configures the various containers that host Wikijump in the local environment. You can find this file in `install/local/dev/`, named `docker-compose.yaml`.
```

Comment:
I would also mention `docker-compose.dev.yaml`, which provides various helpfuls for developing locally (mapping local volumes and such).

Basically, `docker-compose.yaml` is the basic setup describing all the (local) containers and how they can talk to each other, and `docker-compose.dev.yaml` adds various conveniences on top, but is (at least should be) optional from the base one.